migrations/20200822184341-create-user-model.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.sequelize.query(
```

Comment:
I see why I changed email to a varchar. I kept hitting `ERROR: type "citext" does not exist` when I tried to run the migration. Creating this extension first resolved it, but let me know if there's another way we should resolve.

Reply:
Sorry, yes, this is exactly what is needed to enable this support...